python/v3.11.1/wl-build.sh

```diff
@@ -51,7 +51,10 @@ logStatus "Preparing artifacts... "
 mkdir -p ${WASMLABS_OUTPUT}/bin 2>/dev/null || exit 1
 mkdir -p ${WASMLABS_OUTPUT}/usr/local/lib 2>/dev/null || exit 1
 
-cp python.wasm ${WASMLABS_OUTPUT}/bin/python${WASMLABS_RUNTIME:+-$WASMLABS_RUNTIME}.wasm || exit 1
-cp usr/local/lib/python3.11.zip ${WASMLABS_OUTPUT}/usr/local/lib/ || exit 1
+logStatus "Optimizing python binary..."
+wasm-opt -O2 -o ${WASMLABS_OUTPUT}/bin/python${WASMLABS_RUNTIME:+-$WASMLABS_RUNTIME}.wasm python.wasm || exit 1
+
+logStatus "Copying standard libs..."
+cp usr/local/lib/python311.zip ${WASMLABS_OUTPUT}/usr/local/lib/ || exit 1
 
 logStatus "DONE. Artifacts in ${WASMLABS_OUTPUT}"
```

Comment:
Not visible in this diff. Do we want to wrap the `make` in the same way as https://github.com/vmware-labs/webassembly-language-runtimes/blob/8aec06d60932a8beacb85a14d246532b3fb50de0/php/php-7.3.33/wl-build.sh#L34-L43, so that we avoid `clang` calling to `wasm-opt` on its own?